packages/design-tokens/src/tokens/functional/components/testimonial/base.json

```diff
@@ -0,0 +1,15 @@
+{
+    "brand": {
+      "Testimonial": {
+        "quoteSize": {
+          "default": {
+            "value": "var(--brand-text-size-300)"
+          },
+          "large": {
+            "value": "var(--brand-text-size-700)"
+          }
+        }
+      }
+    }
+  }
```

Comment:
The formatting seems off in this file, could you run a prettier fix on it please? Possible why the CI checks are failing too.